pkg/proxy/ipvs/testing/util.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testing
+
+import (
+	utilipset "k8s.io/kubernetes/pkg/util/ipset"
+)
+
+// expectedVirtualServer is the expected ipvs rules with vs and rs
+// vsNum is the expected ipvs virtual server number
+// ip:port protocol is the expected ipvs vs info
+// rs is the real server of this expected vs
+type ExpectedVirtualServer struct {
```

Comment:
Can you please not use acronym's in the comments. For example, instead of saying "vs" and "rs", please say "virtual server" and "real server". I am also okay with adding a comment at the very top that makes it clear that you are using these acronyms.